iac/terraform/aws/auth_cognito/variables.tf

```diff
@@ -0,0 +1,30 @@
+variable "environment" {
+  type        = string
+  description = "Deployment environment"
+}
+
+variable "lambda_runtime" {
+  type        = string
+  description = "Lambda Runtime"
+  default     = "nodejs20.x"
+}
+
+variable "user_pool_name" {
+  type = string
+}
+
+variable "verified_domain_arn" {
+  type = string
+}
+
+variable "table_arn" {
```

Comment:
Only table_arn is not good. Add a better name. Which table arn?

Reply:
Update naming now it's dynamodb_table_arn [Commit](https://github.com/CraftsmenLtd/BloodConnect/commit/28f045306270fdc86a03546bd3b3e08356c4dc74#diff-765ae1142333eea58a06757ec0d1ccea730ed995a95aa9b040fe445375b07459)